server/triffidsapi/trees.py

```diff
@@ -38,6 +46,22 @@ def getTreesBySpecies(parkCode, latinCode):
     print(len(trees))
     return trees
 
+def getNumbUniqueSpeciesByPark(parkCode):
```

Comment:
Just realised this is the same as `getUniqueSpecies` maybe I can just reuse that method and return the length 🤔

Reply:
Keep it DRY.

Also, if you want the Pythonista way of doing it, they use List Comprehensions, like this:
```
with open('./server/data/trees.json') as json_file:
    data = json.load(json_file)
def getTotalNumbTreesByPark(code):
    return sum(r['fields']['site_code'] == code for r in data)
def getUniqueTreesByPark(code):
    res = filter(lambda r: r['fields']['site_code'] == code, data)
    return set([r['fields']['latin_code'] for r in res])
```
but that is no faster and arguably less clear.